Fixtures/content/htaccessContent.php

```diff
@@ -0,0 +1,204 @@
+<?php
```

Comment:
I'm not sure of the reusability of this part either

Reply:
You're likely right. I'll remove it.

Reply:
https://github.com/wp-media/phpunit-wp-rocket/pull/1/commits/18a0db57b8951c5c680d5329e4108ead963f2bb0